files/qsub.pl

```diff
@@ -908,7 +908,27 @@ sub parse_all_resource_list
     my (@resource_list) = @_;
 
     my ($res_opts, $node_opts);
+
+    my %multiplier = (
+        half => 0.5,
+        all  => 1,
+        );
+
+    my $have_numproc = 0;
+    my $tnumproc = 0;
+    if ($ENV{VSC_INSTITUTE_CLUSTER}) {
+        $tnumproc = qx(python -c "from vsc.jobs.pbs.clusterdata import CLUSTERDATA; print CLUSTERDATA['$ENV{VSC_INSTITUTE_CLUSTER}']['NP']" 2>&1);
+        $have_numproc = 1 if !$?;
+    }
+
     foreach my $rl (@resource_list) {
+        if ($have_numproc) {
+            while ($rl =~ m/ppn=(all|half)/) {
+                my $np = $tnumproc * $multiplier{$1};
```

Comment:
needs to become an integer. probably flooring makes most sense, so you can simly use `int()`

Reply:
fixed